odsApi_versioned_docs/version-7.3/technical-articles/upgrading-admin-and-security-to-pg16.md

```diff
@@ -0,0 +1,155 @@
+# Upgrading the Admin and Security Databases to PostgreSQL 16.x
+
+PostgreSQL 16.x is minimum supported version for Ed-Fi ODS / API v7.3. There are
+three available methods for updating instances of the Admin and Security
+database running on earlier versions of PostgreSQL to 16.x:
+
+* Restoring database backups to a PostgreSQL 16.x instance
+
+* Using the [`pg_upgrade`](https://www.postgresql.org/docs/16/pgupgrade.html)
+utility
+
+* Using logical replication
+
+This article provides a high-level overview of upgrading Admin and Security
+databases instances to PostgreSQL 16.x using database backups.
+
+:::info
+
+The details for performing a database upgrade are deployment-specific, and this
+ article describes the process when carried out on a single-server installation
+using version-specific directory names. Detailed information on performing
+version upgrades and additional upgrade methods are available in the official
+PostgreSQL documentation for [Upgrading a PostgreSQL
+Cluster](https://www.postgresql.org/docs/16/upgrading.html#UPGRADING).
+
+:::
+
+## Upgrading using backups
+
+### Step 1. Create backups of the Admin and Security databases
+
+Use the `pg_dump` utility from the currently installed version of PostgreSQL to
+create backups of each database.
+
+<details>
+<summary>Windows examples</summary>
+
+```none
+C:\Program Files\PostgreSQL\<old_version>\bin\pg_dump.exe -U pgadmin -d EdFi_Admin -f EdFi_Admin_backup.sql
+```
+
+```none
+C:\Program Files\PostgreSQL\<old_version>\bin\pg_dump.exe -U pgadmin -d EdFi_Security -f EdFi_Security_backup.sql
+```
+
+</details>
+
+<details>
+<summary>Linux examples</summary>
+
+```none
+/usr/pgsql-<old version>/bin/pg_dump -U pgadmin -d EdFi_Admin --create -f EdFi_Admin_backup.sql
+```
+
+```none
+/usr/pgsql-<old version>/bin/pg_dump --create -U pgadmin -d EdFi_Security --create -f EdFi_Security_backup.sql
+```
+
+</details>
```

Comment:
You may want to consider renaming this page to `.mdx` instead of `.md` and using Tabs here instead of two different `<details>` panels. However, the current approach is also perfectly fine.

Reply:
Tabs would give a better visual presentation, and I will make that change. I will also remove the `--create` flags from these commands since the step below includes manually creating the empty databases during restoration. I was on the fence about using that flag and later decided on the manual database creation step since it gives users a little more flexibility. However, I missed removing it from these commands (and even included it twice in one somehow...)